src/core/tsi/ssl_transport_security.cc

```diff
@@ -1140,6 +1140,10 @@ void tsi_ssl_session_cache_unref(tsi_ssl_session_cache* cache) {
   tsi::SslSessionLRUCache::FromC(cache)->Unref();
 }
 
+size_t tsi_ssl_session_cache_size(tsi_ssl_session_cache* cache) {
+  return tsi::SslSessionLRUCache::FromC(cache)->Size();
```

Comment:
At present this is only used in testing, agreed. However, this is IMO a very natural API to add to the current list. For example, if we intend to clean up the resumption stack during the de-experimentalization work, then likely adding logs for the state/size of the session cache will be a useful part of that, and it would be used there. This is a loosely held opinion so please push back if you don't feel that this is the right place for this API.

Reply:
I'm good with it then - at first I also thought it might be doing a copy in `FromC` which would be an inefficient helper for size, but it looks like it's just a static pointer cast under the hood

Feel free to resolve if this all sounds good to you